.nsprc

```diff
@@ -1,5 +1,7 @@
 {
   "exceptions": [
+    "https://nodesecurity.io/advisories/55",
+
```

Comment:
These are grouped by module, iirc. The top one is **moment**, the two below are **uglify-js**, etc.
Which is also why they aren't in ascending order (your next nit below).